content/blog/posts/2022-o3de-user-survey-report.md

```diff
@@ -0,0 +1,49 @@
+
+{{< image-width src="/images/blog/user-survey-2022/survey-2022.png" width="100%" alt="Mean Ratings Across 3 User Experience Metreics" >}}
+
+We also asked users to rate their experience with the features within the workflow. Based on the data, the Help System across O3DE, the integration with the version control tools, the toolings (e.g. Terrain and Dynamic Vegetation) in the World Building workflow, the Physics System, and the asset management experience (including Asset Browser) received lower satisfaction ratings (ranging from 2.5 to 2.8). The UX performance of these features could be the reason that drove lower ratings in their designated workflows.
+
+#### Using Qualitative Data to Deepen Understanding
+
+While this quantitative survey data can point to areas of overall improvement in O3DE’s user experience, we need qualitative data to inform a deeper understanding of the results to guide the future evolution of the engine’s user experience. During the summer months, qualitative interviews were conducted with 20 community members, focused on the five common workflows users employ when building their content with O3DE: Onboarding Experience, Actor Development, Look Development, Multiplayer, and Engine Extension.
+
+Through these interviews, 138  issues were identified that either block or impact users from accomplishing their tasks. Using this quantitative data to pinpoint and prioritize areas for improvement  and more deeply understand these areas, SIG UI-UX worked with all of the other SIGs to improve these five workflows. You can find the issues tracked on the GitHub [2022 Walk The Engine Issues Dashboard](https://github.com/o3de/o3de/projects/15).
+
+A development branch with the critical issues we identified and fixed will be released in late October, and we will conduct another round of O3DE UI/UX research using this branch towards the end of this year to verify if the improvements meet user expectations.
+
+#### What Other Suggestions Do We Have?
+
+We will continue to address the issues identified in this research to improve the user experience, simplify the onboarding experience, lower the learning curve, deliver greater efficiency to users, and otherwise increase confidence in using O3DE. Simultaneously, we also suggest the related SIGs delve into the following areas and improve their user experience holistically.
```

Comment:
```suggestion
We will continue to address the issues identified in this research to improve the user experience, simplify the onboarding experience, lower the learning curve, deliver greater efficiency to users, and otherwise increase confidence in using O3DE. Simultaneously, we also suggest that the related SIGs delve into the following areas and improve their user experience holistically.
```